lightkurve/correctors/tests/test_sffcorrector.py

```diff
@@ -166,3 +167,10 @@ def test_sff_breakindex():
                  centroid_col=np.random.randn(20),
                  centroid_row=np.random.randn(20), windows=1)
     assert_array_equal(corr.window_points, np.asarray([5, 10]))
+
+
+def test_sff_tess_warning():
+    """SFF is not designed for TESS, so we raise a warning."""
+    lc = TessLightCurve(flux=[1, 2, 3])
+    with pytest.warns(LightkurveWarning, match='not suitable'):
+        corr = SFFCorrector(lc)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [local variable 'corr' is assigned to but never used](https://app.codacy.com/manual/barentsen/lightkurve/pullRequest?prid=4910439)